App crashing on login and skip

**Actual Behaviour**
App crashes when user tries to login or clicks on "Skip".

**Expected Behaviour**
User should be able to login.
**Steps to reproduce it* 
Click on skip on login screen or create a new user and try to login
**LogCat for the issue**

08-31 21:32:10.655 813-813/org.fossasia.susi.ai E/AndroidRuntime: FATAL EXCEPTION: main
                                                                  Process: org.fossasia.susi.ai, PID: 813
                                                                  java.lang.RuntimeException: Unable to start activity ComponentInfo{org.fossasia.susi.ai/org.fossasia.susi.ai.chat.ChatActivity}: java.lang.IllegalArgumentException: ChatMessage is not part of the schema for this Realm
                                                                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3139)
                                                                      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3238)
                                                                      at android.app.ActivityThread.access$1000(ActivityThread.java:197)
                                                                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1681)
                                                                      at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                      at android.os.Looper.loop(Looper.java:145)
                                                                      at android.app.ActivityThread.main(ActivityThread.java:6862)
                                                                      at java.lang.reflect.Method.invoke(Native Method)
                                                                      at java.lang.reflect.Method.invoke(Method.java:372)
                                                                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
                                                                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
                                                                   Caused by: java.lang.IllegalArgumentException: ChatMessage is not part of the schema for this Realm
                                                                      at io.realm.internal.modules.CompositeMediator.getMediator(CompositeMediator.java:172)
                                                                      at io.realm.internal.modules.CompositeMediator.getTableName(CompositeMediator.java:90)
                                                                      at io.realm.RealmSchema.getTable(RealmSchema.java:296)
                                                                      at io.realm.RealmSchema.getSchemaForClass(RealmSchema.java:316)
                                                                      at io.realm.RealmQuery.<init>(RealmQuery.java:138)
                                                                      at io.realm.RealmQuery.createQuery(RealmQuery.java:87)
                                                                      at io.realm.Realm.where(Realm.java:1214)
                                                                      at org.fossasia.susi.ai.data.db.DatabaseRepository.getMessageCount(DatabaseRepository.kt:25)
                                                                      at org.fossasia.susi.ai.chat.ChatPresenter.setUp(ChatPresenter.kt:57)
                                                                      at org.fossasia.susi.ai.chat.ChatActivity.setUpUI(ChatActivity.kt:97)
                                                                      at org.fossasia.susi.ai.chat.ChatActivity.onCreate(ChatActivity.kt:81)
                                                                      at android.app.Activity.performCreate(Activity.java:6536)
                                                                      at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1120)
                                                                      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3092)
                                                                      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3238) 
                                                                      at android.app.ActivityThread.access$1000(ActivityThread.java:197) 
                                                                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1681) 
                                                                      at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                      at android.os.Looper.loop(Looper.java:145) 
                                                                      at android.app.ActivityThread.main(ActivityThread.java:6862) 
                                                                      at java.lang.reflect.Method.invoke(Native Method) 
                                                                      at java.lang.reflect.Method.invoke(Method.java:372) 
                                                                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404) 
                                                                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199) 



**Would you like to work on the issue?**
Yes

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -1,3 +1,8 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'realm-android'
+
 buildscript {
     repositories {
         maven { url 'https://maven.fabric.io/public' }
@@ -7,9 +12,7 @@ buildscript {
         classpath 'io.fabric.tools:gradle:1.+'
     }
 }
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
+
 //apply plugin: 'io.fabric'
 
 repositories {
@@ -17,8 +20,6 @@ repositories {
     mavenCentral()
 }
 
-apply plugin: 'android-apt'
-apply plugin: 'realm-android'
 
 android {
     compileSdkVersion 25
@@ -64,7 +65,7 @@ dependencies {
 
     //butterknife
     compile 'com.jakewharton:butterknife:8.4.0'
-    apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    kapt 'com.jakewharton:butterknife-compiler:8.4.0'
 
     compile "com.android.support:support-v4:$support_lib_version"
 
@@ -95,7 +96,7 @@ dependencies {
     //waiting dots
     compile 'com.github.tajchert:WaitingDots:0.3.2'
     compile('com.crashlytics.sdk.android:crashlytics:2.6.5@aar') {
-        transitive = true;
+        transitive = true
     }
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
 }

--- a/build.gradle
+++ b/build.gradle
@@ -12,8 +12,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:2.3.0'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
-        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
-        classpath "io.realm:realm-gradle-plugin:2.0.2"
+        classpath "io.realm:realm-gradle-plugin:3.5.0"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 
         // NOTE: Do not place your application dependencies here; they belong

